dfget/core/downloader/downloader.go

```diff
@@ -42,7 +42,8 @@ type Downloader interface {
 // the given timeout duration.
 func DoDownloadTimeout(downloader Downloader, timeout time.Duration) error {
 	if timeout <= 0 {
-		logrus.Warnf("invalid download timeout(%.3fs)", timeout.Seconds())
+		logrus.Debugf("invalid download timeout(%.3fs), use default:(%.3fs)",
```

Comment:
I think it is better to use `Warnf`, since it is a useful message for operators.

Reply:
Yes, it's better to use `WARN` level.
But it makes user confused because that this invalid value is set by dfget not specified by user. I will open another pr to fix it.